src/Grid.js

```diff
@@ -3,21 +3,20 @@ import Button from './button';
 
 import './App.css';
 
+const Step = ({ step = [], index, toggleButton }) => (
+  <div className="grid-step">
+    <h1>{index + 1}</h1>
+    {step.map(props => <Button toggleButton={toggleButton} {...props} />)}
```

Comment:
Here's another common pattern: a props passthrough. We can use the spread syntax to just fling whatever params we've got into the component as props.